source/code/plugins/out_oms.rb

```diff
@@ -29,6 +29,11 @@ def initialize
     def configure(conf)
       s = conf.add_element("secondary")
       s["type"] = ChunkErrorHandler::SecondaryName
+
+      # override default run_in_background config
+      if !OMS::BackgroundJobs::enabled.nil?
+        conf['run_in_background'] = OMS::BackgroundJobs::enabled
```

Comment:
you can reassign conf dict? why not just reassign `@run_in_background`

Reply:
If you assign @run_in_background fluentd will reset that value from "conf['run_in_background']".
Look at other fluentd internal plugins, they all use this technique to override configs.

Reply:
yeah after i commented i checked, whoops